### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by olorin**
Don't merge yet, dependent on pending changes in retcon#144.

### CODE REVIEW

File: lib/System/Nagios/Plugin/Retcond.hs [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+data RetconMeters = RetconMeters
+  { _entityMeters           :: Map Text EntityMeters
+  , _serverNumNotifications :: EkgGaugeValue
+  } deriving (Eq, Show)
+
+makeLenses ''RetconMeters
+
+instance FromJSON RetconMeters where
+    parseJSON (Object o) = RetconMeters <$> o .: "datasources"
```
Comment (oswynb) [inline lib/System/Nagios/Plugin/Retcond.hs]: this should probably be entities?

**Comment by tranma** [comment]
you don't need tojson?

File: lib/System/Nagios/Plugin/Retcond.hs (hunk 1 above)
Comment (olorin) [inline lib/System/Nagios/Plugin/Retcond.hs]: Indeed it should.

**Comment by olorin** [comment]
@tranma: don't think so; I have no need to produce JSON, only consume it.

**Pull request merged**